src/containers/Screens/characterInfo/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {ReactElement} from 'react';
+import {useTranslation} from 'react-i18next';
+import {Image, Text, View} from 'react-native';
+import {useSelector} from 'react-redux';
+import {selectCharacter} from 'store/characters/selectors';
+import styles from './styles';
+
+const CharacterInfo = ({route}): ReactElement => {
```

Comment:
Please add type to navigation props - https://reactnavigation.org/docs/typescript/